src/main/java/seedu/contax/model/onboarding/OnboardingStep.java

```diff
@@ -85,14 +87,26 @@ public String getCommand() {
         return command;
     }
 
-    public int getPersonOperation() {
+    public int getOperationId() {
```

Comment:
Local variable name should change if you are going to call it something else.

Reply:
Fixed